test/gtest/ucp/test_ucp_am.cc

```diff
@@ -641,7 +644,9 @@ class test_ucp_am_nbx : public test_ucp_am_base {
         return 0;
     }
 
-    static const uint16_t           TEST_AM_NBX_ID = 0;
+    static const uint16_t           TEST_AM_NBX_ID        = 0;
+    volatile size_t                 m_am_sent_counter     = 0;
+    volatile size_t                 m_am_received_counter = 0;
```

Comment:
remove m_am_received and use m_am_received_counter instead in the base class as well